packages/python3.xlmmacrodeobfuscator.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,19 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = "XLMMacroDeobfuscator"
+  $category = "Office"
+
+  $whlUrl = "https://github.com/DissectMalware/$toolName/releases/download/v0.2.7/$toolName-0.2.7-py3-none-any.whl"
+
+  # Install the wheel file using python3 pip
+  VM-Execute-Custom-Command -name "Install with python3 pip" -cmds "pip install $whlUrl"
```

Comment:
Is there a reason to use `VM-Execute-Custom-Command` instead of `Invoke-Expression`? `VM-Execute-Custom-Command` was introduced to be used in `VM-Apply-Configurations` used to load custom config files. We don't use it in any package at the moment.

I think we should specify the concrete python version as we do in [libraries.python3.vm](https://github.com/mandiant/VM-Packages/blob/main/packages/libraries.python3.vm/tools/chocolateyinstall.ps1) to ensure there are not issues with other possible installed versions in the system.

```suggestion
Invoke-Expression "py -3.10 -m pip install $whlUrl"
```